egs/ifnenit/v1/path.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# path to Kaldi's root directory
+# root=../../tools/kaldi-trunk/
+# root=/export/b01/babak/kaldi/kaldi/
+export KALDI_ROOT=`pwd`/../../..
+
+export LD_LIBRARY_PATH=/home/dpovey/libs:$KALDI_ROOT/src/chainbin:/usr/local/lib:$LD_LIBRARY_PATH
+
+export PATH=$KALDI_ROOT/tools/srilm/bin/:$KALDI_ROOT/tools/srilm/bin/i686-m64/:$KALDI_ROOT/src/bin:$KALDI_ROOT/src/chainbin:$KALDI_ROOT/src/chain:$KALDI_ROOT/src/nnet3bin:$KALDI_ROOT/src/lmbin:$KALDI_ROOT/tools/openfst/bin:$KALDI_ROOT/src/fstbin/:$KALDI_ROOT/src/gmmbin/:$KALDI_ROOT/src/featbin/:utils:$KALDI_ROOT/src/fgmmbin:$KALDI_ROOT/src/sgmm2bin:$KALDI_ROOT/src/nnetbin:$KALDI_ROOT/src/nnet2bin:$KALDI_ROOT/src/sgmmbin:$KALDI_ROOT/src/lm:$KALDI_ROOT/src/latbin:$PATH 
+# path to the directory in which the subset of RM corpus is stored
+
+export LC_ALL=en_US.utf8
+export LC_LOCALE_ALL=C
+
+export MONO_GAUSS="40000"
```

Comment:
These variables should not be here. They should be in run.sh. Actually, it's better not to use any variables and instead use the values in the corresponding commands.